### ISSUE TITLE
Response Headers not generated

### ISSUE THREAD

**Issue by wspurgin** [kminami/apib2swagger#46]
Looks like Response Headers that aren't "Content-Type" are not output to the swagger output. https://github.com/kminami/apib2swagger/commit/a55845f45e728cfb402024d9d40bd9b4afb496be

For example, take the following blueprint:
```apib
FORMAT: 1A9

# Test

Test API

## Test Collection [/tests]
The collection of test stuff

### Create a Test [POST]

+ Attributes
  + name (string) - The name of the test thing

+ Request (application/json)

            {
                "name": "bar"
            }

+ Response 201 (application/json)

    + Headers

            Location: /tests/1

    + Body

                {
                    "name": "bar",
                    "created_at": "2019-06-19T15:20:51.620Z",
                    "url": "/tests/1"
                }
```

I expect the swagger response section to include the `Location` header and value.

It seems like you could reuse the `swaggerHeader` method used to populate Request headers (other than content-type and authorization) to populate the Response headers.

As of v1.9.2, the above template outputs:

```javascript
{
    "swagger": "2.0",
    "info": {
        "title": "Test",
        "version": "",
        "description": "Test API"
    },
    "paths": {
        "/tests": {
            "post": {
                "responses": {
                    "201": {
                        "description": "Created",
                        "headers": {},
                        "examples": {
                            "application/json": {
                                "name": "bar",
                                "created_at": "2019-06-19T15:20:51.620Z",
                                "url": "/tests/1"
                            }
                        }
                    }
                },
                "summary": "Create a Test",
                "description": "",
                "tags": [],
                "parameters": [
                    {
                        "name": "body",
                        "in": "body",
                        "schema": {
                            "type": "object",
                            "properties": {
                                "name": {
                                    "type": "string",
                                    "description": "The name of the test thing"
                                }
                            }
                        }
                    }
                ],
                "produces": [
                    "application/json"
                ]
            }
        }
    },
    "definitions": {
        "Test Collection": {}
    },
    "securityDefinitions": {},
    "tags": []
}
```

**Comment by wspurgin** [comment]
Looks like it would need a little tweaking because the HeaderObject for 2.0 looks like this: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#headerObject

**Comment by kminami** [comment]
Thank you for a request and sugestions.
I have implemented this at v1.10.0.
Please try it!

**Comment by wspurgin** [comment]
Looks good thanks @kminami
